dev-docs/show-multiformat-ads.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: Show Multiformat Ads with Prebid.js
+description: Show Multiformat Ads with Prebid.js
+pid: 0
+is_top_nav: yeah
+top_nav_section: dev_docs
+nav_section: prebid-multiformat
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Show Multiformat Ads with Prebid.js
+{: .no_toc }
+
+This page has instructions for showing multiformat ads using Prebid.js.
+
+Multiformat ads allow you to declare multiple media types on a single ad unit.  For example, you can set up one ad unit that could show a banner, native, or outstream video ad, depending on which had the highest bid. 
```

Comment:
"An ad unit is said to be multi-format if it supports at least two of the "banner", "native" or "video" media types."

Reply:
Much nicer formulation!  Updated in 08d65cc